### PR TITLE
Add Author endpoint for testing.

### PR DESCRIPTION
**Pull request by michaellibucha**

### CODE REVIEW

**Review by obber — APPROVED** [review]
LGTM!

File: services/api-server/src/author/dto/author.dto.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  @ApiProperty({ type: String })
+  readonly lastName: string;
+
+  @ApiProperty({ type: String })
+  readonly handle: string;
+}
+
+export class CreateAuthorDto {
+  @ApiProperty({ type: String, required: false })
+  firebaseUid: string;
```
[the remark this answers is not in the document]
Comment (michaellibucha) [reply services/api-server/src/author/dto/author.dto.ts:17]: The issue here was that I was passing an object of type CreateAuthorDto to the Author Model to create it in the DB. Do you think I should copy the data over to a differently typed object before I add it to the DB and leave firebaseUid out of the DTO entirely?

[the remark this answers is not in the document]
Comment (michaellibucha) [reply services/api-server/src/author/dto/author.dto.ts]: I figured it out. I'm still figuring out where I can just use type inference, but I was able to construct a new object when calling the model

File: apps/web/src/features/author/components/author-info.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      <Stack>
+        <CircularProgress />
+      </Stack>
+    );
+  }
+
+  if (data?.author) {
+    return (
+      <Stack>
+        <Card key={`${data.author.firstName}-${data.author.lastName}`}>
```
[the remark this answers is not in the document]
Comment (michaellibucha) [reply apps/web/src/features/author/components/author-info.tsx]: Removed

**Pull request merged**